agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -258,6 +261,18 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
               return props;
             })
         .addPropertiesCustomizer(new AiConfigCustomizer())
+        .addLogRecordProcessorCustomizer(
+            (logRecordProcessor, configProperties) -> {
+              if (firstLogRecordProcessor.getAndSet(false)) {
+                // hack to register log record processors in front of the batch log processor
```

Comment:
lgtm, but this comment doesn't make sense to me can we update it to point to your github issue on opentelemetry-java?